charts/cp-schema-registry/templates/deployment.yaml

```diff
@@ -33,6 +33,10 @@ spec:
       {{- end }}
       {{- end }}
     spec:
+      securityContext:
+      {{- if .Values.securityContext }}
+{{ toYaml .Values.securityContext | indent 8 }}
+      {{- end}}
```

Comment:
If no securityContext, the generated yaml would be empty.
So, I think it will be better with the `securityContext` attribute **inside** the`if`, so something like that:
```
      {{- if .Values.securityContext }}
      securityContext: {{ toYaml .Values.securityContext | nindent 8 }}
      {{- end}}
```
(I've also used the nident here, which is another topic)